data/lua/on_event.lua

```diff
@@ -21,20 +21,23 @@ wesnoth.game_events.on_event = function(eventname)
 end
 
 
-local function on_event(eventname, arg1, arg2)
+---Handle an event
```

Comment:
Looks like "register an event handler" rather than actually handling the event here.

Reply:
Good point.